xbmc/GUIInfoManager.cpp

```diff
@@ -5188,6 +5210,50 @@ std::string CGUIInfoManager::GetItemLabel(const CFileItem *item, int info, std::
         stereoMode = CStereoscopicsManager::Get().NormalizeStereoMode(item->GetVideoInfoTag()->m_streamDetails.GetStereoMode());
       return stereoMode;
     }
+  case LISTITEM_ACTOR:
```

Comment:
This whole segment could be made shorter by falling through all the cases, then grab the EPG tag and do a new switch which determines the method to return.